### DISCUSSION TITLE
Use encapsulated browsers instead of pop-ups

### DISCUSSION THREAD

**Discussion by AlinaWang2015** [cefsharp/CefSharp#4797] [General]
CefSharp Version
96.0.180.0

Operating System
Windows 10

Architecture
x86

.Net Version
.Net 6

Implementation
WPF

Reproduction Steps
I use ChromiumWebBrowser to encapsulate a FormBrowser. Now there is a scenario: use FormBrowser to load a web page A, and the user clicks on the hyperlink of web page A, and pop-up window B needs to be loaded. However, I used the following method to instead of pop-up window, but the new window did not work.

```
function ReDealPop(string targetUrl, string targetFrameName) {
  FormBrowser formBrowser = new FormBrowser();
  formBrowser.BrowserUrl = targetUrl;
  formBrowser.BrowserTitleName = targetFrameName;
  formBrowser.IsPop = true;
  formBrowser.ParentBrowser = this;
}
```

**Comment by amaitland** [comment]
> CefSharp Version
> 96.0.180.0

Not a supported version 

> However, I used the following method to instead of pop-up window, but the new window did not work.

It's impossible to say from the code. What does `did not work` mean?

You can host the `Popup` in a `ChroimiumWebBrowser` instance using the following example  https://github.com/cefsharp/CefSharp/blob/cefsharp/124/CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs#L129
